jumpscale/servers/threebot/threebot.py

```diff
@@ -282,7 +282,11 @@ def get(self, package_name):
             return Package(path=package_path, default_domain=self.threebot.domain, default_email=self.threebot.email)
 
     def get_packages(self):
```

Comment:
that won't work in case if you registered outside package right?

Reply:
As per @abom all paths for external packages will be listed under j.packages.__path__